src/MassTransit.QuartzIntegration.Tests/Recurring_Specs.cs

```diff
@@ -104,6 +104,8 @@ public MySchedule()
 
                 StartTime = DateTime.Now + TimeSpan.FromSeconds(3);
                 EndTime = StartTime + TimeSpan.FromSeconds(7);
+
+                Description = "my description";

```

Comment:
Generally you shouldn't need to mock much in this codebase. Most things are composable. Given that, I'm a bit unsure of how to test this as well. What's the point of the description? Just provides the ability to track it down in the DB? It's not exposed to the messages in any way?